font-src/glyphs/letter/latin/s.ptl

```diff
@@ -41,6 +42,15 @@ glyph-block Letter-Latin-S : begin
 			alsoThru.g2 0.5 0.5 [widths.center ess]
 			g4.down.mid (deltaX + RightSB - Width - SOBot) (bot + archDepth) [widths.rhs gap]
 
+	define [SCurlyTail sw] : begin
+		local fine : AdviceStroke2 3 4 XH
+		local rinner : ((XH * 0.5) - fine * 2 - sw * 0.5) / 3
+		local m1 : RightSB - SOBot - sw * HVContrast
```

Comment:
Could try decrease this so the right terminal of the tail won't hit the body of `s`.